src/lib.rs

```diff
@@ -18,6 +18,10 @@ pub enum Instr {
     Div(Addr, Addr, Addr),
     /// a = b % c
     Rem(Addr, Addr, Addr),
+    /// Jumps program execution by n instructions
+    Jump(i8),
+    /// Jumps program execution by n instructions if a is non zero, else it jumps by m instructions
```

Comment:
We have a proper boolean type (the `B` value), so we shouldn't be working in terms of “nonzero.”